zipkin4net/Criteo.Profiling.Tracing.UTest/T_SpanState.cs

```diff
@@ -0,0 +1,62 @@
+﻿using NUnit.Framework;
+
+namespace Criteo.Profiling.Tracing.UTest
+{
+    [TestFixture]
+    public class T_SpanState
+    {
+        private const long traceId = 1;
+        private const long traceIdHigh = 0;
```

Comment:
same nit on ordering, won't repeat again. feel free to go your way and I understand lexicographically this looks right